ui/scripts/templates.js

```diff
@@ -311,6 +311,7 @@
                                     },
                                     // Direct Download - End
 
+                                    //XenServer only (starts here)
```

Comment:
what do you mean 'XenServer *only*'?

Reply:
@DaanHoogland I've changed comments to make more sense